external-crates/move/crates/move-analyzer/src/completion.rs

```diff
@@ -305,12 +310,24 @@ fn call_completion_item(
     });
 
     let method_name = method_name_opt.unwrap_or(function_name);
+    let (insert_text, insert_text_format) = if inside_use {
```

Comment:
Instead of threading this boolean all the way down, would it make sense to rewrite the completion item for the `use` case or similar?

Reply:
> Instead of threading this boolean all the way down, would it make sense to rewrite the completion item for the use case or similar?

Not sure I understand this one... Is the idea to have two versions of this function, one for `use` and one for the other case? If the idea is to re-do the `CompletionItem` struct then it's not possible as it comes from `lsp-types` crate and is part of the LSP spec